test/extract/transpile/svelte-wrap.spec.js

```diff
@@ -0,0 +1,26 @@
+const { promises: fsPromise } = require("fs");
+const { expect } = require("chai");
+const wrap = require("../../../src/extract/transpile/svelte-wrap")(
+  require("../../../src/extract/transpile/typescript-wrap")(true)
+);
+
+describe("svelte transpiler", () => {
+  it("tells the svelte transpiler is available", () => {
+    expect(wrap.isAvailable()).to.equal(true);
+  });
+
+  it("'transpiles' svelte", async () => {
+    const observedPromise = fsPromise
+      .readFile("./test/extract/transpile/fixtures/svelte.svelte", "utf8")
+      .then((pSourceCode) => wrap.transpile(pSourceCode));
```

Comment:
The `expect` will likely fail on windows (because of line-end shenanigans). To prevent that other wrapper unit tests use `normalise-newline` on both the output of the read file and that of the wrap.transpile before passing it to the expect.